frontstage/controllers/conversation_controller.py

```diff
@@ -25,7 +25,7 @@ def _get_session():
 
 
 def get_conversation(thread_id):
-    logger.debug('Attempting to retrieve thread', thread_id=thread_id)
+    logger.info('Attempting to retrieve thread', thread_id=thread_id)
```

Comment:
I'd probably change to read "Retrieving conversation thread" to be clear about what sort of thread we're talking about. Also less wordy :)

Reply:
Done